instrumentation/resources/library/src/test/java/io/opentelemetry/instrumentation/resources/internal/DeclarativeConfigTest.java

```diff
@@ -56,7 +57,8 @@ void endToEnd() {
               assertThat(attributeKeys).contains("os.description");
               assertThat(attributeKeys).contains("os.type");
               // ProcessResourceComponentProvider
-              assertThat(attributeKeys).contains("process.command_line");
+              assertThat(attributeKeys)
+                  .contains(java8 ? "process.command_line" : "process.command_args");
```

Comment:
I am curious why it doesn't influence the CI pipeline running here before. I don't seem to find any adjustment that you've made to this test in this PR. Does it not run here by default?

Reply:
see https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/13225/files#diff-472d337fbc448ae293f03a5a87ce9925b8b414728b564a89ea4d21b166ecbda9